projects/kit/directives/skeleton/skeleton.directive.ts

```diff
@@ -39,8 +42,9 @@ class TuiSkeletonStyles {}
 export class TuiSkeleton implements OnChanges {
     private animation?: Animation;
     private readonly el = tuiInjectElement();
-    private readonly duration =
-        inject(TUI_ANIMATIONS_SPEED) * TUI_ANIMATIONS_DEFAULT_DURATION * 2;
+    private readonly duration = tuiGetDuration(
+        inject(TUI_ANIMATIONS_SPEED) * TUI_ANIMATIONS_DEFAULT_DURATION * 2,
```

Comment:
You do not need `TUI_ANIMATIONS_DEFAULT_DURATION`, it is already there in `tuiGetDuration`. What was intended here, I believe, it to have duration twice longer than regular animations. So you need to change this to `tuiGetDuration(inject(TUI_ANIMATION_SPEED) * 2`